### DISCUSSION TITLE
Pause at inventory

### DISCUSSION THREAD

**Discussion by ACGaming** [ACGaming/UniversalTweaks#208] [Tweak Suggestions]
Makes singleplayer sessions pause while in inventory

**Comment by Primitive-Human** [comment]
I really wonder how this could be implemented without breaking many mods.

Also consider that some mods have items that continue working while in the inventory screen like some sort of timer in the NBT, like the redstone tools in pyrotech, or tool/armor chargers using capacitors in the inventory, or with wireless recharging blocks like those in EnderIO.

**Comment by ACGaming** [comment]
From a technical perspective, this is going to be identical to the pause screen, so no ticking or simulation at all. This might be off-putting in most scenarios but could be a handy feature for special modpack cases.

**Comment by jchung01** [comment]
Looks like the functionality now exists with this mod [Flexipause](https://www.curseforge.com/minecraft/mc-mods/flexipause).
